src/main/java/uk/gov/laa/ccms/caab/bean/validators/proceedings/ProceedingFurtherDetailsValidator.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.laa.ccms.caab.bean.validators.proceedings;
+
+import org.springframework.stereotype.Component;
+import org.springframework.validation.Errors;
+import uk.gov.laa.ccms.caab.bean.proceeding.ProceedingFlowFormData;
+import uk.gov.laa.ccms.caab.bean.proceeding.ProceedingFormDataFurtherDetails;
+import uk.gov.laa.ccms.caab.bean.proceeding.ProceedingFormDataProceedingDetails;
+import uk.gov.laa.ccms.caab.bean.validators.AbstractValidator;
+
+/**
+ * Validator the further details provided by proceeding flow forms.
+ */
+@Component
+public class ProceedingFurtherDetailsValidator extends AbstractValidator {
+
+  /**
+   * Determines if the Validator supports the provided class.
+   *
+   * @param clazz The class to check for support.
+   * @return {@code true} if the class is assignable from
+   *         {@link uk.gov.laa.ccms.caab.bean.proceeding.ProceedingFlowFormData},
+   *         {@code false} otherwise.
+   */
+  @Override
+  public boolean supports(final Class<?> clazz) {
+    return ProceedingFlowFormData.class.isAssignableFrom(clazz);
+  }
+
+  /**
+   * Validates the further details in the
+   * {@link uk.gov.laa.ccms.caab.bean.proceeding.ProceedingFlowFormData}.
+   *
+   * @param target The object to be validated.
+   * @param errors The Errors object to store validation errors.
+   */
+  @Override
+  public void validate(final Object target, final Errors errors) {
+    final ProceedingFlowFormData formData = (ProceedingFlowFormData) target;
+    final ProceedingFormDataFurtherDetails furtherDetails = formData.getFurtherDetails();
```

Comment:
Pass in the ProceedingFormDataFurtherDetails directly, as per ProceedingMatterTypeDetailsValidator

Reply:
see comment aboce